iron-dropdown.js

```diff
@@ -1,13 +1,25 @@
 /**
 @license
 Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
-This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
-The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
-The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+This code may only be used under the BSD style license found at
+http://polymer.github.io/LICENSE.txt The complete set of authors may be found at
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
 */
-/*TODO remove this import */
+import '@polymer/polymer/polymer-legacy.js';
+// TODO: remove this import
```

Comment:
No, this shouldn't be relevant anymore. I think it's just here because importing iron-dropdown.html would make `Polymer.IronDropdownScrollManager` available as a global but that's not something the 3.x version does, so there's no compatibility gain from having it.